README.md

```diff
@@ -86,76 +123,95 @@ proxy.register(require('fastify-reply-from'), {
 })
 ```
 
-#### cacheURLs
+#### `cacheURLs`
+
+The number of parsed URLs that will be cached. Default: `100`.
 
-The number of parsed URLs that will be cached. Default: 100.
+#### `keepAliveMsecs`
 
-#### keepAliveMsecs
+**(Deprecated)** Defaults to 1 minute (`60000`), passed down to [`http.Agent`][http-agent] and
+[`https.Agent`][https-agent] instances. Prefer to use [`http.agentOptions`](#http) instead.
 
-Defaults to 1 minute, passed down to [`http.Agent`][http-agent] and
-[`https.Agent`][https-agent] instances.
+#### `maxSockets`
 
-#### maxSockets
+**(Deprecated)** Defaults to `2048` sockets, passed down to [`http.Agent`][http-agent] and
+[`https.Agent`][https-agent] instances. Prefer to use [`http.agentOptions`](#http) instead.
 
-Defaults to 2048 sockets, passed down to [`http.Agent`][http-agent] and
-[`https.Agent`][https-agent] instances.
+#### `maxFreeSockets`
 
-#### rejectUnauthorized
+**(Deprecated)** Defaults to `2048` free sockets, passed down to [`http.Agent`][http-agent] and
+[`https.Agent`][https-agent] instances. Prefer to use [`http.agentOptions`](#http) instead.
 
-Defaults to `true`, passed down to [`https.Agent`][https-agent] instances.
+#### `rejectUnauthorized`
+
+**(Deprecated)** Defaults to `false`, passed down to [`https.Agent`][https-agent] instances.
```

Comment:
Should not emit a warning when deprecated features are used?

Reply:
Sorry, I didn't really get what you mean there. 

Do you mean we should add some deprecation warning in code? Or add some docs documenting that the library would not emit a warning when a deprecated feature is used?

Reply:
Yes, I mean to add a (`once`) warning in the code that the developer will see:

https://github.com/fastify/fastify/blob/master/lib/warnings.js

Reply:
Gotcha. Will do so!

Reply:
If you could do it in a following PR it would be appreciated since this PR has been merged 👍